dask_jobqueue/core.py

```diff
@@ -450,7 +454,7 @@ def __init__(
         scheduler_cls=Scheduler,  # Use local scheduler for now
         # Options for both scheduler and workers
         interface=None,
-        protocol="tcp://",
+        protocol=None,
```

Comment:
to me this default did not make immediate sense. However one could revert this line I guess

Reply:
Fine by me.